packages/api-explorer/src/scenes/TypeTagScene/TypeTagScene.spec.tsx

```diff
@@ -74,30 +86,28 @@ describe('TypeTagScene', () => {
     ).toHaveLength(2)
   })
 
-  test('it filters methods by operation type', async () => {
+  test('it pushes filter to URL on toggle', async () => {
```

Comment:
what about a test asserting that the right tag is selected when the scene is rendered with a filter value already in the store?

Reply:
In a scenario where the URL has no verb filter param, the store has a selected verb filter, and we are rendering the page: upon execution of the reconciliation hook, because there is no verb filter param in the URL, the page will render having selected the 'ALL' filter. Is this the assertion you want me to make?